R/loo-glossary.R

```diff
@@ -62,20 +64,53 @@
-#' sampling is not able to give reliable estimate. If `k<0.5`, then the
-#' corresponding component of `elpd_loo` is estimated with high accuracy.
-#' If `0.5<k<0.7` the accuracy is lower, but still ok. If `k>0.7`,
-#' then importance sampling is not able to provide useful estimate for that
-#' component/observation. Pareto k is also useful as a measure of influence of
-#' an observation. Highly influential observations have high k values. Very high
-#' k values often indicate model misspecification, outliers or mistakes in data
-#' processing. See Section 6 of Gabry et al. (2019) for an example.
+#' sampling is not able to give reliable estimate. Pareto smoothing stabilizes
+#' importance sampling and guarantees finite variance estimate with a
+#' cost of some bias.
+#'
+#' The diagnostic threshold for Pareto \eqn{k} depends on sample size
+#' \eqn{S} (sample size dependent threshold was introduced by Vehtari
+#' et al., 2022, and before that fixed thresholds of 0.5 and 0.7 were
+#' recommended). For simplicity, `loo` package uses the nominal sample
+#' size \eqn{S}  when computing the sample size specific
+#' threshold. This provides an optimistic threshold if the effective
+#' sample size is less than 2200, but even then if ESS/S>1/2 the difference
+#' is usually negligible. Thinning of MCMC draws can be used to improve
+#' the ratio ESS/S.
+#'
+#' * If \eqn{k < min(1 - 1 / log10(S), 0.7)}, where \eqn{S} is the
+#'   sample size PSIS estimate and the corresponding Monte
+#'   Carlo standard error estimate are reliable.
+#'
+#' * If \eqn{1 - 1 / log10(S) <= k < 0.7}, PSIS estimate and the
+#'   corresponding Monte Carlo standard error estimate are not
+#'   reliable, but increasing (effective) sample size \eqn{S} above
+#'   2200 may help (this will increase the sample size specific
+#'   threshold \eqn{(1-1/log10(2200)>0.7} and then the bias specific
+#'   threshold 0.7 dominates).
+#'
+#' * If \eqn{0.7 <= k < 1}, PSIS estimate and the corresponding Monte
+#'   Carlo standard error have large bias and are not reliable. Increasing
+#'   sample size may reduce the variability in \eqn{k} estimate, which
+#'   may result in lower \eqn{k} estimate, too.
+#'
+#' * If \eqn{k \geq 1}{k >= 1}, the target distribution is estimated to
+#'   have non-finite mean. PSIS estimate and the corresponding Monte
+#'   Carlo standard error are not well defined. Increasing sample size
+#'   may reduce the variability in \eqn{k} estimate, which
+#'   may result in lower \eqn{k} estimate, too.
+#' 
+#' Pareto \eqn{k} is also useful as a measure of influence of an
+#' observation.  Highly influential observations have high \eqn{k}
+#' values. Very high \eqn{k} values often indicate model
+#' misspecification, outliers or mistakes in data processing. See
+#' Section 6 of Gabry et al. (2019) for an example.
 #'
 #' \subsection{Interpreting `p_loo` when Pareto `k` is large}{
 #' If `k > 0.7` then we can also look at the `p_loo` estimate for
```

Comment:
```
#' If `k > 0.7` then we can also look at the `p_loo` estimate for
```

Should this threshold be the sample size specific one? Or is it ok to have it specified as 0.7 here

Reply:
I guess the sample size specific one would be more consistent with the changes in this PR. @avehtari? 

I think some places in the FAQ at https://github.com/stan-dev/loo/blob/master/vignettes/online-only/faq.Rmd also probably need to be updated.

Reply:
I'll work on FAQ and there will be a lot of updates there